pkg/lightning/backend/local/local.go

```diff
@@ -152,7 +152,37 @@ type File struct {
 	// isImportingAtomic is an atomic variable indicating whether this engine is importing.
 	// This should not be used as a "spin lock" indicator.
 	isImportingAtomic atomic.Uint32
+<<<<<<< HEAD
 	mutex             sync.Mutex
+=======
```

Comment:
merge conflict

Reply:
need other cherry-picked pr to be merged first. 😂